providers/azure/auth.go

```diff
@@ -0,0 +1,57 @@
+package azure
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/adal"
+	"github.com/Azure/go-autorest/autorest/azure"
+)
+
+const (
+	credentialsPath = "/.azure/credentials.json"
+)
+
+// ToJSON returns the passed item as a pretty-printed JSON string. If any JSON error occurs,
+// it returns the empty string.
+func ToJSON(v interface{}) (string, error) {
+	j, err := json.MarshalIndent(v, "", "  ")
+	return string(j), err
+}
```

Comment:
I dont think this is being used anymore. It was back when I was testing out azure auth so ill just remove it.